src/pages/case-study/copenhagen-atomics.tsx

```diff
@@ -0,0 +1,364 @@
+      title={title}
+      image="/img/pages/case-study/copenhagen-atomics/banner.png"
+    >
+      <section
+        className={clsx(
+          seCss.section,
+          seCss["section--center"],
+          juCss.jumbotron,
+        )}
+      >
+        <div className={juCss.jumbotron__summary}>
+          <div className={juCss.jumbotron__header}>
+            <Button
+              href="https://www.copenhagenatomics.com/?utm_source=questdb"
+              variant="plain"
+            >
+              <img
+                alt="Copenhagen Atomics logo"
+                className={juCss.jumbotron__logo}
+                height={45}
+                src={logos["copenhagen-atomics"].src}
+                width={100}
+              />
+            </Button>
+            <span className={juCss.jumbotron__name}>Case study</span>
+          </div>
+          <h1 className={seCss.section__title}>Copenhagen Atomics</h1>
+          <p
+            className={clsx(
+              seCss.section__subtitle,
+              juCss.jumbotron__description,
+            )}
+          >
+            {description}
+          </p>
+        </div>
+
+        <div className={juCss.jumbotron__banner}>
+          <Image
+            alt="Copenhagen Atomics's major footprints in Thailand comprises over 4 million sqm of net leasable area, 60 malls, 2400 retail stores, 1000 food outlets, and 53 hotels."
+            description={
+              <>
+                An overview of the benefits brought by the Copenhagen Atomics
+                Waste Burner.{" "}
+                <a
+                  href="https://www.copenhagenatomics.com/potential/"
+                  target="_blank"
+                  rel="noreferrer noopener"
+                >
+                  Image source
+                </a>
+                .
+              </>
+            }
+            src="/img/pages/case-study/copenhagen-atomics/overview.png"
+            width={777}
+            height={350}
+          />
+        </div>
+      </section>
+
+      <section className={clsx(seCss.section, seCss["section--odd"])}>
+        <div className={clsx(seCss["section--inner"], ouCss.outcome__wrapper)}>
+          {values.map(({ icon, description }, index) => (
+            <p key={index} className={ouCss.outcome}>
+              <img
+                alt={icon.alt}
+                className={ouCss.outcome__icon}
+                src={icon.src}
+              />
+              {description}
+            </p>
+          ))}
+        </div>
+      </section>
+
+      <section className={seCss.section}>
+        <div
+          className={clsx(
+            "markdown",
+            seCss["section--inner"],
+            seCss["section--column"],
+          )}
+        >
+          <h3>Introduction</h3>
+          <p className="font-size--large">
+            Founded by a group of scientists and engineers in 2014, Copenhagen
+            Atomics aims to address one of the most burning issues for our
```

Comment:
Sorry...I forgot to add a hyperlink to their site for "Copenhagen Atomics"
https://www.copenhagenatomics.com/
One day I will get it right:
```suggestion
            Founded by a group of scientists and engineers in 2014,{" "}
            <a href="https://www.copenhagenatomics.com">Copenhagen Atomics</a>{" "} aims to address one of the most burning issues for our
```

Reply:
all good, don't worry, I'll add it